mitodl/open-discussions
# refactor course pages searchbox and header image

#### Pre-Flight checklist


#### What are the relevant tickets?

part of #1997 

#### What's this PR do?

this basically implements the search bar that is shown in the mocks for the course home page and the course search page.

#### How should this be manually tested?

check out the course home page (`/courses`) and the search page (`/courses/search/`) and make sure what we're showing in the top part looks somewhat like the mocks!

#### Screenshots (if appropriate)
![courseasdffnffcoursesffj](https://user-images.githubusercontent.com/6207644/62398158-ec19de80-b545-11e9-9b63-83aaeaba962a.png)
![courseasdffnff](https://user-images.githubusercontent.com/6207644/62398160-ecb27500-b545-11e9-95ca-75fbf5bcfd05.png)
